helm_deploy/hmpps-interventions-service/templates/cronjob-migrate-referral-details-to-changelog-report.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: migrate-referral-details-to-changelog
+spec:
+  schedule: "30 0 * * *"
```

Comment:
_not a blocker_
We discussed maybe gating this or making it on-demand -- I'm looking up what we can use for that